pkg/clusterresource/controller.go

```diff
@@ -357,7 +358,9 @@ func (c *controller) Sync(ctx context.Context) error {
 	logger.Debugf(ctx, "Running an invocation of ClusterResource Sync")
 
 	// Prefer to sync projects most newly created to ensure their resources get created first when other resources exist.
-	projects, err := c.db.ProjectRepo().ListAll(ctx, descCreatedAtSortParam)
+	projects, err := c.db.ProjectRepo().List(ctx, repositoriesInterfaces.ListResourceInput{
+		SortParameter: descCreatedAtSortParam,
```

Comment:
nit: as an optimization can we add a filter for active projects (now that we can filter!)

Reply:
Sure we can provider a filter here. Filtering out non-active projects is the default behavior when no filter provided, to keep backward compatibility.

Reply:
This is addressed by https://github.com/lyft/flyteadmin/pull/142/commits/bf478e5936fdfebc23d530334ae03df9028c7664

Reply:
oh you're right, sorry i thought it was done in the manager layer for some reason. thank you for adding regardless, my bad!